packages/elements/src/number-field/__test__/number-field.step.test.js

```diff
@@ -220,4 +220,43 @@ describe('number-field/Step', function () {
       await expectValues(el, [-1, -2, -3, -4, -5], UP, -1.5);
     });
   });
+
+  describe('Support long press', function () {
+    let el;
+    const dispatchLongTapEvent = async (el, wait) => {
+      setTimeout(() =>
+        el.dispatchEvent(
+          new Event('tapstart', {
+            bubbles: true
+          })
+        )
+      );
+      await oneEvent(el, 'tapstart');
+      if (wait) {
+        await aTimeout(wait);
+      }
+      setTimeout(() =>
+        el.dispatchEvent(
+          new Event('tapend', {
+            bubbles: true
+          })
+        )
+      );
+      await oneEvent(el, 'tapend');
+    };
+
+    beforeEach(async function () {
```

Comment:
judge from this test file context, I think we don't need `beforeEach()` here.

Reply:
Removed.